language/libra-vm/src/libra_vm.rs

```diff
@@ -457,6 +457,10 @@ impl LibraVM {
         txn_data.sender = account_config::CORE_CODE_ADDRESS;
         txn_data.max_gas_amount = GasUnits::new(std::u64::MAX);
 
+        let registered_currencies =
```

Comment:
Was wondering if we should persist this in LibraVM for performance.

Reply:
It's a good question! At least for now I would say "no" for two reasons, but more than open to reconsidering:
1. The registered currencies aren't core to the operation of the VM (at least in my view). So they shouldn't be held as part of the VM state.
2. I'm not sure what happens in terms of the cache invalidation if a currency is registered. Does the VM need to be restarted in this case?